src/genie/libs/parser/iosxe/tests/test_show_ap.py

```diff
@@ -0,0 +1,239 @@
+import unittest
+from unittest.mock import Mock
+
+from genie.metaparser.util.exceptions import SchemaEmptyParserError
+from genie.libs.parser.iosxe.show_ap import ShowApSummary
+
+
+# ===============================
+# Unit test for 'show ap summary'
+# ===============================
+class TestShowApSummary(unittest.TestCase):
+    """Unit test for 'show ap summary'"""
+
+    maxDiff = None
+    empty_output = {'execute.return_value': ''}
+    golden_parsed_output1 = {
```

Comment:
Let's have a key for AP Name like `a121-cap22`. (and I removed top level key `ap_summary`. technically we don't need)
```
{
    "ap_neighbor_count": 149,
    "ap_name": {              # ADDED
        "a121-cap22": {
            "slots_count": 2,
            (snip)
        },
        (snip)
    }
}
```

Reply:
@tahigash, This was updated.